core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1153,6 +1157,8 @@ object KafkaConfig {
       .define(MetadataMaxRetentionBytesProp, LONG, Defaults.MetadataMaxRetentionBytes, null, HIGH, MetadataMaxRetentionBytesDoc)
       .define(MetadataMaxRetentionMillisProp, LONG, LogConfig.DEFAULT_RETENTION_MS, null, HIGH, MetadataMaxRetentionMillisDoc)
       .define(MetadataMaxIdleIntervalMsProp, INT, Defaults.MetadataMaxIdleIntervalMs, atLeast(0), LOW, MetadataMaxIdleIntervalMsDoc)
+      .defineInternal(BrokerServerMaxStartupTimeMs, LONG, Defaults.BrokerServerMaxStartupTimeMs, atLeast(0), MEDIUM, BrokerServerMaxStartupTimeMsDoc)
```

Comment:
Seems like we could get away with one internal config. Maybe `process.max.startup.time.ms`?

Reply:
yeah